services/ui-src/src/components/layout/Header.test.tsx

```diff
@@ -16,44 +18,58 @@ describe("<Header />", () => {
       render(headerComponent);
     });
 
-    test("Navigation, Logo, Help and Menu is visible on Header", () => {
+    test("Logo, Help and Menu is visible on Header", () => {
       const header = screen.getByRole("navigation");
       expect(header).toBeVisible();
-      expect(screen.getByAltText("HCBS logo")).toBeVisible();
-      expect(screen.getByAltText("Help")).toBeVisible();
+      // find img elements
+      expect(screen.getByRole("img", { name: "HCBS logo" })).toHaveAttribute(
+        "alt",
+        "HCBS logo"
+      );
+      expect(screen.getByRole("img", { name: "Help" })).toHaveAttribute(
+        "alt",
+        "Help"
+      );
+      expect(screen.getByRole("img", { name: "Account" })).toHaveAttribute(
+        "alt",
+        "Account"
+      );
       expect(screen.getByAltText("Arrow down")).toBeVisible();
     });
 
-    test("Renders My Account menu and is clickable", () => {
-      render(
-        <MenuOption
-          text={"My Account"}
-          icon={"arrows/icon_arrrow_down.svg"}
-          altText={"Arrow down"}
-        />
-      );
-
-      const menuButton = screen.getByRole("button", { name: /my account/i });
+    test("When My Account menu is clicked, it expands", async () => {
+      // Find button
+      const menuButton = screen.getByRole("button", {
+        name: "my account",
+        expanded: false,
+      });
       expect(menuButton).toBeInTheDocument();
-
-      userEvent.click(menuButton);
+      // Click and expand
+      await act(async () => {
+        await userEvent.click(menuButton);
+      });
```

Comment:
Same suggestion here (2/3)

```suggestion
      await userEvent.click(menuButton);
```